addons/im_livechat/static/tests/livechat_channel_info_list.test.js

```diff
@@ -69,3 +75,84 @@ test("editing livechat note is synced between tabs", async () => {
     await document.querySelector(".o-livechat-ChannelInfoList textarea").blur(); // Trigger the blur event to save the note
     await waitFor(".o-livechat-ChannelInfoList textarea:value('Updated note')", { target: tab1 }); // Note should be synced with bus
 });
+
+test("livechat status is displayed in discuss sidebar", async () => {
+    const pyEnv = await startServer();
+    const userId = pyEnv["res.users"].create({ name: "James" });
+    pyEnv["res.partner"].create({
+        name: "James",
+        user_ids: [userId],
+    });
+    const countryId = pyEnv["res.country"].create({ code: "be", name: "Belgium" });
+    const guestId = pyEnv["mail.guest"].create({
+        name: "Visitor #20",
+    });
+    const channelId = pyEnv["discuss.channel"].create({
+        anonymous_name: "Visitor #20",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ guest_id: guestId }),
+        ],
+        country_id: countryId,
+        channel_type: "livechat",
+        livechat_operator_id: serverState.partnerId,
+        livechat_status: "waiting",
+    });
+    await start();
+    await openDiscuss(channelId);
+    await waitFor(".o-livechat-ChannelInfoList button.active", { text: "Waiting for customer" });
+    await waitFor(
+        ".o-mail-DiscussSidebar-item span[title='Waiting for customer'] i.fa-circle.o-warning"
+    );
+    await click(".o-livechat-ChannelInfoList button", { text: "Looking for help" });
+    await waitFor(".o-livechat-ChannelInfoList button.active", { text: "Looking for help" });
+    await waitFor(".o-mail-DiscussSidebar-item span[title='Looking for help'] i.fa-circle.o-info");
```

Comment:
- I'd prefer assert title and not icon or `span`, to keep selector as short as needed for test.
- `.o-livechat-ChannelInfoList` => `.o-mail-Discuss-panelContainer` (better idea which UI element this refers to, and also I plan to change to rename this classname because it doesn't exactly follows BEM naming)